maintenance/wikia/createUser.php

```diff
@@ -41,7 +43,7 @@
 }
 
 # Insert the account into the database
-$user->addToDatabase();
+//$user->addToDatabase();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)